Http/ResponseType.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace DUCK.Http
+{
+	public enum ResponseType
+	{
+		Unknown,
+		Successful,
```

Comment:
`Success`?

I wonder if we should use a noun rather than an adjective.

Reply:
Agreed

Reply:
+1